modules/gdscript/gdscript_parser.cpp

```diff
@@ -148,7 +148,9 @@ GDScriptParser::GDScriptParser() {
 	// Networking.
 	register_annotation(MethodInfo("@rpc", PropertyInfo(Variant::STRING, "mode"), PropertyInfo(Variant::STRING, "sync"), PropertyInfo(Variant::STRING, "transfer_mode"), PropertyInfo(Variant::INT, "transfer_channel")), AnnotationInfo::FUNCTION, &GDScriptParser::rpc_annotation, varray("", "", "", 0), true);
 
+#ifdef DEBUG_ENABLED
```

Comment:
This is correct, but the definition of `is_ignoring_warning` in the header should also move in a `DEBUG_ENABLED` block.

I don't see how this would solve a crash though?